src/nile/nre.py

```diff
@@ -4,11 +4,15 @@
 from nile.core.call_or_invoke import call_or_invoke
 from nile.core.compile import compile
 from nile.core.deploy import deploy
+from nile.core.utils import _get_installed_plugins, skip_click_exit
 
 
 class NileRuntimeEnvironment:
     """The NileRuntimeEnvironment exposes Nile functionality when running a script."""
 
+    for name, object in _get_installed_plugins().items():
```

Comment:
I believe that if we pass the plugins dict in a `__init__` function then it's possible to also add tests for this. It requires a change in `run` to load the plugins and then instantiate `nre`.